modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -3986,7 +3969,7 @@ Maximum number of concurrent append entry requests sent by the leader to one fol
 
 === raft_max_recovery_memory
 
-Max memory that can be used for reads in Raft recovery process by default 15% of total memory.
+Maximum memory that can be used for reads in Raft recovery process by default 15% of total memory.
```

Comment:
Seems a but unclear.

Is the max memory value 15% of total memory then? 
Maybe:
Maximum memory that can be specified for reads in a Raft recovery process. By default, this value is 15% of total memory.